model-optimizer/mo/middle/passes/fusing/fuse_linear_ops.py

```diff
@@ -84,6 +91,36 @@ def _fuse_mul(graph: Graph, node: Node, fuse_nodes: list, backward: bool = True)
         ch_dim = weights_port.data.get_attr('output_channel_dim' if backward else 'input_channel_dim')
         shape = np.array([weights_port.data.get_shape()[ch_dim]])
 
+        group = fuse_node.soft_get('group', 1)
+        if group != 1 and shape[0] % group != 0:
+            shape *= group
```

Comment:
We have agreed to not perform this weights permutation and continue this case without performing any changes. We just need to update the condition in line 95 to be more precise.

@GlebKazantaev , @jane-intel could you provide the correct condition?

Reply:
I just braced reshaping of the const with try-except, this will fix situations when const cannot be reshaped according to weights shape.